kodyrouse/eventbrite-interview
# Added /LabelValue shared component

Closes #

## Background

> Explain the context and why you're making that change. What is the
problem you're trying to solve? In some cases there is not a problem
and this can be thought of being the motivation for your change.

## Modification

> Describe the modifications you've done.

## Result

> What will change as a result of your pull request? Note that sometimes
this section is unnecessary because it is self-explanatory based on
the solution.

## How to test

> What are the steps necessary to independently confirm that the PR
is correct in staging or production? What steps did you do to manually
smoke test or verify the functionality? This may not apply to all
PRs.

## :ghost: GIF

![](url)
